Source/cursor.cpp

```diff
@@ -535,7 +535,7 @@ void CheckCursMove()
 			pcursmonst = -1;
 			cursPosition = { mx, my };
 		}
-		if (pcursmonst != -1 && Monsters[pcursmonst].isPlayerMinion()) {
+		if ((pcursmonst != -1 && Monsters[pcursmonst].isPlayerMinion()) || (pcurs == CURSOR_HEALOTHER || pcurs == CURSOR_RESURRECT)) {

```

Comment:
Thanks :)

Maybe use `IsAnyOf`
```suggestion
		if ((pcursmonst != -1 && Monsters[pcursmonst].isPlayerMinion()) || IsAnyOf(pcurs, CURSOR_HEALOTHER, CURSOR_RESURRECT)) {
```

Reply:
Sure thing, that is a nice constexpr